apps/volontulo/tests/views/test_offers.py

```diff
@@ -8,7 +8,7 @@
 from django.test import Client
 from django.test import TestCase
 
-from apps.volontulo.models import Offer
+from apps.volontulo.models import Offer, Badge
```

Comment:
I suggest to use separate imports. It helps if somthings broke up and throw line number.